docs/poisoning_object_detection.md

```diff
@@ -1,12 +1,11 @@
 # Object Detection Poisoning
 
-Object Detection Poisoning (using the YOLOv3 model) requires [PytorchYolo](https://github.com/eriklindernoren/PyTorch-YOLOv3) (with its accompanying dependencies).  Armory currently does not include this in the base docker image, neither does it maintain a separate docker image just to support this.
-
-To run, please install via ```pip install .[yolo]``` and run using ```--no-docker```.
+Object Detection Poisoning (using the YOLOv3 model) requires [PytorchYolo](https://github.com/eriklindernoren/PyTorch-YOLOv3) (with its accompanying dependencies).  Armory currently does not include this in the base docker image, however, support does exist in the `twosixarmory/yolo` image.
 
+To run, install the `yolo` dependencies via ```pip install .[yolo]```.
```

Comment:
Just to clarify that you don't need to pip install if you use the docker image, could I suggest:

Object Detection Poisoning (using the YOLOv3 model) requires [PytorchYolo](https://github.com/eriklindernoren/PyTorch-YOLOv3) (with its accompanying dependencies). Armory includes this in the twosixarmory/yolo docker image; it can also be installed via pip install .[yolo] and run using --no-docker.